server/profile.py

```diff
@@ -26,20 +28,23 @@ def createProfile(name, birthday, bio, gender, education, interests, genderPrefe
 
     return {"response": "Something went wrong creating profile"}
 
-def updateProfile(name, birthday, bio, gender, education, interests, genderPreference, maxDistance, age, id):
+def updateProfile(name, birthday, bio, gender, education, interests, genderPreference, maxDistance, age, id, image):
     try:
         connection = connectToDB()
         if(connection != False):
+            
+            image_url = saveImage(image, id)
             cursor = connection.cursor(buffered=True)
             sql = '''UPDATE Profile SET firstname = %s, age = %s, description = %s, gender = %s, \
                 workplace = %s, interests = %s, genderPreference = %s, maxDistance = %s, Birthday = %s \
-                 WHERE userId = %s;'''
+                , picture = %s WHERE userId = %s;'''
             values = (name, age, bio, gender, education,
-                      interests, genderPreference, maxDistance, birthday, id)
+                      interests, genderPreference, maxDistance, birthday, image_url, id)
```

Comment:
possibly make image_url -> imageURL to be consistent with the one from createProfile?